src/cpp/core/r_util/RSourceIndex.cpp

```diff
@@ -418,6 +418,87 @@ void testThatCallIndexer(const RTokenCursor& cursor,
    }
 }
 
+void stringAfterRoxygenIndexer(const RTokenCursor& cursor,
+                               const IndexStatus& status,
+                               bool isReadOnlyFile, 
+                               RSourceIndex* pIndex)
+{
+   if (!cursor.isType(RToken::STRING))
+      return;
+   
+   RTokenCursor clone = cursor.clone();
+   if (!clone.bwdOverWhitespace())
+      return;
+
+   if (!clone.moveToPreviousToken())
+      return;   
+
+   if (!clone.isType(RToken::COMMENT))
+      return;
+
+   if (!boost::algorithm::starts_with(clone.content(), "#'"))
+      return;
+
+   RSourceItem item(
+      RSourceItem::Roxygen,
+      cursor.contentAsUtf8(),
+      std::vector<RS4MethodParam>(), 
+      status.count(RToken::LBRACE),
+      cursor.row() + 1, 
+      cursor.column() + 1, 
+      isReadOnlyFile
+   );
+
+   pIndex->addSourceItem(item);
+}
+
+void nameRoxygenIndexer(const RTokenCursor& cursor,
+                        const IndexStatus& status,
+                        bool isReadOnlyFile, 
+                        RSourceIndex* pIndex)
+{
+   if (!cursor.isType(RToken::ID))
+      return;
+
+   if (!cursor.contentEquals(L"NULL"))
+      return;
+
+   RTokenCursor clone = cursor.clone();
+   
+   if (!clone.bwdOverWhitespace())
+      return;
+
+   while (clone.moveToPreviousToken()) 
+   {
+      if (!clone.isType(RToken::COMMENT))
+         return;
+
+      if (!boost::algorithm::starts_with(clone.content(), "#'"))
```

Comment:
I think it's valid to include multiple `#` markers in a roxygen comment, e.g. `##' some text`.